packages/gatsby-source-graphcms/README.md

```diff
@@ -0,0 +1,48 @@
+# gatsby-source-graphql
+
+Source plugin for pulling data into [Gatsby](https://github.com/gatsbyjs) from a GraphQL endpoint.
+
+## How to use
+*In your gatsby config...*
+```javascript
+plugins: [
+  /*
+   * Gatsby's data processing layer begins with “source”
+   * plugins. Here the site sources its data from a graphql endpoint
+   */
+  {
+    resolve: `gatsby-source-graphcms`,
+    options: {
+      endpoint: `graphql_endpoint`,
+      token: `graphql_token`,
+    },
+  }
+],
+```
+I suggest either using a `.env` file or setting environment variables to access ala `process.env.GATSBY_GRAPHQL_ENDPOINT`, `process.env.GATSBY_GRAPHQL_TOKEN` so theres no endpoint and token on git ;)
```

Comment:
Is there a special way to use process.env for plugins? Does the gatsby-config not have access to webpack's definitions at execution? I haven't tried it out yet, admittedly, so I will when I get home tonight.